src/components/patch-assignment-form/components/form-buttons.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { Dispatch, SetStateAction } from "react";
+
+import { Patch } from "@mtfh/common/lib/api/patch/v1";
+
+export const CancelReassignmentButton = ({
+  onClick,
+}: {
+  onClick: Function;
+}): JSX.Element => {
+  return (
+    <button
+      data-testid="cancel-reassignment-button"
+      className="govuk-button lbh-button"
+      style={{ marginTop: 0 }}
+      onClick={(e) => {
+        e.preventDefault();
```

Comment:
I think you can get rid of `e.preventDefault()` if you set the type attribute to "button".

Reply:
Thanks, did that in 9bd2a8ec33cd97fbd1c4326f914ccddcf29f6d37